BookReader/BookReader.js

```diff
@@ -2796,7 +2796,8 @@ BookReader.prototype.flipRightToLeft = function(newIndexL, newIndexR) {
 };
 
 BookReader.prototype.setMouseHandlers2UP = function() {
-    this.setClickHandler2UP( this.prefetchedImgs[this.twoPage.currentIndexL],
+    var currentIndexL = this.prefetchedImgs[this.twoPage.currentIndexL];
```

Comment:
I feel like the changes made in this function could be rewritten to avoid a lot of duplicate code. I don't know if I can easily describe the approach, but I can add a commit on this branch demonstrating it.